cmd/node/node.go

```diff
@@ -503,6 +503,8 @@ func (app *SpacemeshApp) initServices(nodeID types.NodeId, swarm service.Service
 
 		return true
 	}
+
+	app.Config.HARE.Limit = 2 * (app.Config.Hdist + 1)
```

Comment:
I believe that it should be with out the +1

Reply:
Removed. We will support up to 3 concurrent hare instances.